src/backend/updates.rs

```diff
@@ -80,7 +80,12 @@ fn post_upgrade() {
 }
 
 #[allow(clippy::all)]
-fn sync_post_upgrade_fixtures() {}
+fn sync_post_upgrade_fixtures() {
+    mutate(|state| {
+        // Distribute revenue from the last auction: https://dashboard.internetcomputer.org/transaction/6b959af7ad051d15f78d818bd27b53ccc51f84e825e0ee9a940582a20860be23?index=13096034
+        state.distribute_revenue_from_icp(6907960000);
```

Comment:
I would vote for not fixing it and keeping it in TAGGR mainly because I don't know how to verify the account addresses in the transaction above and more importantly I don't want to spend time doing that :smile:

Reply:
Sorry, I don't understand what you mean. Keeping what in TAGGR? The TAGGR tokens auctioned were minted to bidders. But now we have the ICP from the auction sitting in the Treasury that we need to distribute.

You can verify that we need to distribute 69 ICP easily using the Taggr Dashboard only (https://6qfxa-ryaaa-aaaai-qbhsq-cai.icp0.io/#/dashboard):

1. We know we auctioned with 115 TAGGR.
2. The dashboard says, the established market price is `60069300` e8s (115 * 60069300 = ~69 ICP).
3. Click on the Treasury link on the dashboard which leads you to the treasury ICP account and make sure you see the transaction linked above being executed at `2024-07-26, 19:02:55 UTC` which is the UTC distribution time. 

We of course could not restore anything, but I feel a bit like a betrayer after advertising the new revenue source in the [last proposal](https://6qfxa-ryaaa-aaaai-qbhsq-cai.icp0.io/#/post/1385856) 😅

Reply:
Thanks for the detailed explanation! Now I can follow and verify. The retrofix lgtm!